### PR TITLE
Payflow: Pass OrderDesc field

### PR DESCRIPTION
**Pull request by curiousepic**
Also adds tests for similar fields.

Remote (10 failures for unrelated account-specific features):
28 tests, 104 assertions, 10 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
64.2857% passed

Unit:
46 tests, 208 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

**Comment by nfarve** [comment]
Was there a reason to not add to the remote tests?

**Comment by curiousepic** [comment]
@nfarve Mostly since these fields don't come back in responses, but it doesn't hurt of course. Added.

File: test/remote/gateways/remote_merchant_warrior_test.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def setup
     @credit_card = credit_card(
       '5123456789012346',
       :month => 5,
-      :year => 17,
+      :year => 18,
```
Comment (nfarve) [inline test/remote/gateways/remote_merchant_warrior_test.rb]: Did you mean to include this in this PR?

Comment (curiousepic) [reply test/remote/gateways/remote_merchant_warrior_test.rb]: Whoops, no, ty.

Comment (curiousepic) [reply test/remote/gateways/remote_merchant_warrior_test.rb]: fixed

**Pull request closed without merging**